xonsh/ptk/completer.py

```diff
@@ -48,6 +49,11 @@ def reserve_space(self):
         except AttributeError:
             #new layout to become default
             window = cli.application.layout.children[1].content
+
+        # This is the layout for ptk 1.0
+        if prompt_toolkit.__version__.startswith("1."):
```

Comment:
You should probably use the `xonsh.shell.promt_tookit_version()` function https://github.com/scopatz/xonsh/blob/master/xonsh/shell.py#L31 as a safety. Also maybe you could put the above tr-except in an else block  after the if here.  Just to make the logic clean.